tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/ClientCredentialsTests.cs

```diff
@@ -115,14 +115,33 @@ public async Task WithClientClaims_OverrideClaims_TestAsync(Cloud cloud, RunOn r
             await RunClientCredsAsync(cloud, CredentialType.ClientClaims_MergeClaims).ConfigureAwait(false);
         }
 
-        private async Task RunClientCredsAsync(Cloud cloud, CredentialType credentialType)
+        [TestMethod]
+        [DataRow(Cloud.Public, RunOn.NetCore)]
+        // [DataRow(Cloud.Arlington)] - cert not setup
+        public async Task WithClientClaims_SendX5C_ExtraClaims_TestAsync(Cloud cloud, RunOn runOn)
+        {
+            runOn.AssertFramework();
+            await RunClientCredsAsync(cloud, CredentialType.ClientClaims_ExtraClaims, sendX5C:true).ConfigureAwait(false);
+        }
+
+        [TestMethod]
+        [DataRow(Cloud.Public, RunOn.NetFx)]
+        [DataRow(Cloud.Adfs, RunOn.NetCore)]
+        // [DataRow(Cloud.Arlington)] - cert not setup
+        public async Task WithClientClaims_SendX5C_OverrideClaims_TestAsync(Cloud cloud, RunOn runOn)
```

Comment:
Integration tests are not asserting anything. Why not adding WithClientClaims as a unit test?

Reply:
Added new unit test